src/java.base/share/classes/javax/net/ssl/TrustManagerFactory.java

```diff
@@ -69,16 +68,13 @@ public class TrustManagerFactory {
      * the value of the {@code ssl.TrustManagerFactory.algorithm}
      * security property to the desired algorithm name.
      *
-     * @see java.security.Security security properties
```

Comment:
it looks like unnecessary API changes here

Reply:
Why do you think it is an API change? Technically it's not related, but minor cleanup changes like this are ok in my opinion.

Reply:
Yes, you are right, javadoc is the same. So, not entirely correct about API.
But the same exact place in the KeyManagerFactory is not modified. I do not know the exact policy about references in the javadoc. I prefer to have a full class name if not in the same package. But, from my point of view, it is better to have consistent changes in similar places.

Reply:
Yes, I think it makes sense to revert it for consistency. Fixed.